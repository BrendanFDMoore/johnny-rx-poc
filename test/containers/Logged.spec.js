import React from 'react';
import { mount } from 'enzyme';

import Logged from 'containers/Logged';

function setup() {
  const props = {
    dispatch: () => {},
    location: {}
  };

  return mount(<Logged {...props} />);
}

describe('Logged', () => {
  const wrapper = setup(true);

  it('should be a StatelessComponent', () => {
    expect(wrapper.instance().constructor.name).toBe('StatelessComponent');
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
