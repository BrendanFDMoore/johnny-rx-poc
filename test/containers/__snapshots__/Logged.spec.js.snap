exports[`Logged should render properly 1`] = `"<div class=\"app__logged app__route\"><a href=\"#load\" class=\"btn btn-primary\">Load</a><a href=\"#cancel\" class=\"btn btn-primary\">Cancel</a><div class=\"app__container\"></div></div>"`;
